apps/site/lib/site_web/views/schedule_view.ex

```diff
@@ -362,22 +362,24 @@ defmodule SiteWeb.ScheduleView do
     "/fares/" <> route_type <> "-fares"
   end
 
-  @spec single_trip_fares(Route.t()) :: [{String.t(), String.t() | iolist}]
+  @spec single_trip_fares(Route.t() | Route.type_int() | String.t()) :: [
```

Comment:
Everywhere this is used seems to be passing in a `%Route{}`, I'm curious why change this?

Reply:
The immediate reason was the [test case](https://github.com/mbta/dotcom/blob/master/apps/site/test/site_web/views/schedule_view_test.exs#L449) for this function calls it using a string and was breaking due to a recent change. But it does seem like this function is only called from a single place, so I'm ok with changing the test instead and leaving the more restrictive spec.

Reply:
[Fixed](https://github.com/mbta/dotcom/pull/1396/commits/0d41fa2ef55aed55e4f63c68d1ea37a51f3c2092)